library/std/src/io/mod.rs

```diff
@@ -1399,16 +1404,15 @@ pub trait Write {
     ///
     /// # Notes
     ///
-    ///
-    /// Unlike `io::Write::write_vectored`, this takes a *mutable* reference to
-    /// a slice of `IoSlice`s, not an immutable one. That's because we need to
+    /// Unlike [`write_vectored`], this takes a *mutable* reference to
```

Comment:
nit: the `[write_vectored]: Self::write_vectored` on line 1403 is hard to see because it's both above where you use the link and hidden in the diff. Could you move it to just before `Examples` instead?

Reply:
That's the result of a find and replace I didn't check thoroughly enough, I'll move it